packages/cli/tests/watch.test.js

```diff
@@ -33,4 +34,28 @@ describe('preact', () => {
 
 		server.close();
 	});
+
+	it('should hot reload and prerender without a development server', async () => {
+		let dir = await create('default');
+		await watch(dir, undefined, undefined, false);
+
+		const initialContent = await fs.readFile(
+			resolve(dir, './build/bundle.js'),
+			'utf8'
+		);
+
+		let header = resolve(dir, './src/components/header/index.js');
+		let original = await fs.readFile(header, 'utf8');
+		let update = original.replace('<h1>Preact App</h1>', '<h1>Test App</h1>');
+		await fs.writeFile(header, update);
+
+		await sleep(1500); // wait for a rebuild
+		const updatedContent = await fs.readFile(
+			resolve(dir, './build/bundle.js'),
+			'utf8'
+		);
+
+		expect(updatedContent).not.toEqual(initialContent);
+		expect(updatedContent).not.toContain('Preact App');
+	});
```

Comment:
I'm not 100% on this test, if there's a better way to go about this certainly let me know. Without starting and stopping the Polka server that's there I believe this is the best option? Just does a simple comparison/look through the bundle output. 

Very possible there's a better way to do this though.